index.bs

```diff
@@ -3254,8 +3254,12 @@ credential.
         with more than one [=user account=] at the [=[RP]=].
 
     :   <dfn>displayName</dfn>
-    ::  A [=human palatability|human-palatable=] name for the [=user account=], intended only for display. For example, "Alex Müller" or "田中倫". The
-        [=[RP]=] SHOULD let the user choose this, and SHOULD NOT restrict the choice more than necessary.
+    ::  A [=human palatability|human-palatable=] name for the [=user account=], intended only for display.
+        [=Clients=] may display this value to help users differentiate between multiple sub-accounts for a
+        given [=user account=] for situations in which every credential uses the same value for
+        {{PublicKeyCredentialEntity/name}}. Examples of suitable values for this identifier include,
+        "Alex Müller" or "田中倫". The [=[RP]=] SHOULD let the user choose this, and SHOULD NOT restrict
+        the choice more than necessary.
```

Comment:
In prettty much every case I've ever seen, this gets set to the same value as user.name. Is it worth mentioning that is a perfectly acceptable thing to do?

Reply:
@sbweeden I tried updating the text accordingly, let me know what you think. @emlun I'm curious to hear if the change I made today requires refinement, or if your current approval still stands.

Reply:
See reply in https://github.com/w3c/webauthn/pull/1932#discussion_r1281744220

Reply:
@sbweeden I'd like to mark this resolve. Can you please review, and if everything looks good to you approve?

Reply:
lgtm - resolving

Reply:
Thanks for taking another look. Can you "Approve" the PR too when you get a chance?